src/components/pages/templateLanding.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { useEffect, useCallback, useMemo } from "react";
+
+import { useTranslation } from "react-i18next";
+import { useSearchParams } from "react-router-dom";
+
+import { howToBuildAutomation, whatIsAutoKitteh } from "@src/constants";
+import { ModalName } from "@src/enums/components";
+import {
+	HiddenIntegrationsForTemplates,
+	IntegrationForTemplates,
+	Integrations,
+	IntegrationsMap,
+} from "@src/enums/components/connection.enum";
+import { useCreateProjectFromTemplate } from "@src/hooks";
+import { TemplateMetadata } from "@src/interfaces/store";
+import { useModalStore, useTemplatesStore, useProjectStore } from "@src/store";
+
+import { Button, IconButton, IconSvg, Spinner, Typography, Frame, Loader } from "@components/atoms";
+import { WelcomeVideoModal } from "@components/organisms/dashboard";
+import { ProjectTemplateCreateModal } from "@components/organisms/dashboard/templates/tabs";
+
+import { ProjectsIcon } from "@assets/image";
+import { CirclePlayIcon, PipeCircleDarkIcon } from "@assets/image/icons";
+
+export const TemplateLanding = () => {
+	const { t: tWelcome } = useTranslation("dashboard", { keyPrefix: "topbar" });
+	const { t } = useTranslation("dashboard", { keyPrefix: "welcome" });
+	const { openModal } = useModalStore();
+	const { createProjectFromAsset, isCreating } = useCreateProjectFromTemplate();
+	const { fetchTemplates, sortedCategories, isLoading } = useTemplatesStore();
+	const { projectsList } = useProjectStore();
+	const [searchParams] = useSearchParams();
+	const assetDir = searchParams.get("assetDirectory");
+
+	const projectNamesSet = useMemo(() => new Set(projectsList.map((project) => project.name)), [projectsList]);
+	const selectedTemplate = useMemo(() => {
+		return sortedCategories?.reduce<TemplateMetadata | undefined>((found, category) => {
+			return found || category.templates.find((template) => template.assetDirectory === assetDir);
+		}, undefined);
+	}, [sortedCategories, assetDir]);
+
+	useEffect(() => {
+		fetchTemplates();
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, []);
+
+	const handleCreateClick = useCallback(() => {
+		if (!selectedTemplate?.assetDirectory) return;
+
+		if (projectNamesSet.has(selectedTemplate.assetDirectory)) {
+			openModal(ModalName.templateCreateProject);
+			return;
+		}
+		createProjectFromAsset(selectedTemplate.assetDirectory);
```

Comment:
The project title looks much more sympathetic than the `asseDirectory`.
Let's create the project with the project title:
```suggestion
		createProjectFromAsset(selectedTemplate.assetDirectory, selectedTemplate.title);
```

When you do that, please check if the name exists by the title on line 50.

Reply:
added selectedTemplate.title